.github/ISSUE_TEMPLATE/Bug_report.yml

```diff
@@ -0,0 +1,107 @@
+name: "Bug Report"
+description: Use this template to report a bug
+labels: ["bug"]
+body:
+  - type: markdown
+    id: example
+    attributes:
+      value: >
+        Thanks for reporting.
+        
+        - Make sure you are able to reproduce this issue on the latest released version of [Azure CLI](https://learn.microsoft.com/en-us/cli/azure/install-azure-cli).
+        
+        - Please search the existing issues to see if there has been a similar issue filed.
+        
+        - If you want to discover useful tips on how to avoid common pitfalls and use the Azure CLI successfully, please refer to [Tips for using the Azure CLI successfully](https://learn.microsoft.com/en-us/cli/azure/use-cli-effectively).
+  - type: textarea
+    id: desc
+    attributes:
+      label: Describe the bug
+      description: >
+        Please describe the bug.
+      placeholder: >
+        A clear and concise description of what the bug is.
+    validations:
+      required: true
+  - type: textarea
+    id: cmd
+    attributes:
+      label: Related command
+      description: Please provide the related commands and parameters with az {command} if you can, so that we can quickly route to the related team for help.
+      placeholder: e.g., `az vm create --image`
```

Comment:
Can we just keep `az vm create` without parameters? Just like what `az feedback` generates.

Reply:
May I ask why remove parameters? Wouldn't it be more helpful to provide more detailed commands?

Reply:
In my opinion, this field is a category label. If its value is too detailed, it will lose the ability to classify.

Reply:
In fact, it will not affect the classification, it will still match the `az vm` pattern.